packages/jsActions/native-mobile-resources/src/camera/TakePictureAdvanced.ts

```diff
@@ -36,147 +98,260 @@ export async function TakePictureAdvanced(
     pictureQuality?: PictureQuality,
     maximumWidth?: Big,
     maximumHeight?: Big
-): Promise<boolean> {
```

Comment:
Why does it return a boolean or an object? You can't handle that in a nanoflow, right?

Reply:
I'm not sure why this was added. I removed it